src/domain/service/note.ts

```diff
@@ -117,6 +117,15 @@ export default class NoteService {
     return updatedNote;
   }
 
+  /**
+   * Delete note parent relation
+   *
+   * @param noteId - id of current note
+   */
+  public async deleteParentRelationById(noteId:NoteInternalId): Promise<boolean> {
+    return this.noteRelationsRepository.deleteParentRelationById(noteId);
+  }
```

Comment:
```suggestion
  public async deleteNoteRelationById(noteId:NoteInternalId): Promise<boolean> {
    return this.noteRelationsRepository.deleteNoteRelationById(noteId);
  }
```

Reply:
We already have method with similar name ("deleteNoteRelationsByNoteId"), but it deletes all relations with this note. I think this naming will be a little confusing